tests/index.spec.js

```diff
@@ -915,4 +915,24 @@ describe('Record obt data, check', () => {
     expect(obtData.payee_fio_address).to.be.a('string')
     expect(obtData.payee_fio_address).to.equal(testFioAddressName2)
   })
+
+  it(`getFee for BurnFioAddress`, async () => {
+    const result = await fioSdk.genericAction('getFeeForBurnFioAddress', {
+        fioAddress: newFioAddress
+    })
+
+    expect(result).to.have.all.keys('fee')
+    expect(result.fee).to.be.a('number')
+  })
+
+  it(`Burn fio address`, async () => {
+    const result = await fioSdk.genericAction('burnFioAddress', {
+        fioAddress: newFioAddress,
+        maxFee: defaultFee
+    })
+
+    expect(result).to.have.all.keys('status', 'fee_collected')
+    expect(result.status).to.be.a('string')
+    expect(result.fee_collected).to.be.a('number')
+  })
```

Comment:
move this to the `describe('Testing generic actions', () => {`
there is no `newFioAddress` here as I understand
the same for `tests/testnet.spec.js`